config/config.go

```
@@ -180,6 +199,17 @@ func (c *Config) RmCommand(name string) error {
 	return nil
 }
 
+func (c *Config) Update(name string, input *Command) error {
+
+	for pos, cmd := range c.Commands {
+		if cmd.Name == name {
+			c.Commands[pos] = input
+			return nil
+		}
+	}
+	return errors.New("")
```

Can we return an error string here?